Led.cpp

```diff
@@ -8,26 +8,29 @@ Led::Led(unsigned pin) : PIN(pin) {
 }
 
 void Led::setBrightness(unsigned percent) {
-    percent *= 2.55;
-    //	analogWrite(led, percent);
-    brightness = percent;
+    if (ledPower) {
+        brightness = percent *= 2.55;
```

Comment:
sort of, but there is no reason to assign the value to `percent`. Just multiply it and assign to `brightness`